statsmodels/tsa/arima_model.py

```diff
@@ -421,9 +423,7 @@ def __init__(self, endog, order, exog=None, dates=None, freq=None,
                  missing='none'):
         super(ARMA, self).__init__(endog, exog, dates, freq, missing=missing)
         # GH 2575
-        _endog = endog if hasattr(endog, 'ndim') else np.asarray(endog)
-        if (_endog.ndim == 2 and _endog.shape[1] != 1) or _endog.ndim > 2:
-            raise ValueError('endog must be 1-d or 2-d with 1 column')
+        array_like(endog, 'endog')
```

Comment:
should this be `endog = array_like(endog, 'endog')`?

Reply:
This is just a check. endog isn't used below

Reply:
All of the data handling is done somewhere in base.